src/main.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { RouterProvider } from "react-router-dom";
+import { router } from "./router";
+import "./index.css";
+
+ReactDOM.createRoot(document.getElementById("root")!).render(
+  <React.StrictMode>
```

Comment:
u should add here ErrorBoundary and display the error you're throwing in case of error

Reply:
It's currently caught using the react-router. Is that how we do it or is ErrorBoundary needed anyway?